### ISSUE TITLE
Automated performance testing

### ISSUE THREAD

**Issue by BrynCooke** [apollographql/router#78]
> * Blocked by https://github.com/apollographql/router-perf-testing/issues/11

Automated daily performance tests.
First cut does not need to be complex, create something that we can add to.

**Comment by abernix** [comment]
@BrynCooke What's the difference between this and #58 ?

**Comment by abernix** [comment]
Duplicate of #58, which is closed.
